tracer/src/Datadog.Trace/Iast/Settings/IastSettings.cs

```diff
@@ -63,7 +63,7 @@ public IastSettings(IConfigurationSource source, IConfigurationTelemetry telemet
                                .AsString(DefaultRedactionValuesRegex);
         RegexTimeout = config
                                 .WithKeys(ConfigurationKeys.Iast.RegexTimeout)
-                                .AsDouble(200, val1 => val1 is > 0).Value;
+                                .AsDouble(200, val1 => val1 is >= 0).Value;
```

Comment:
so in `TracerSettings` we do: 
ObfuscationQueryStringRegexTimeout = config
                                                .WithKeys(ConfigurationKeys.ObfuscationQueryStringRegexTimeout)
                                                .AsDouble(200, val1 => val1 is > 0).Value;
dont we want to harmonize the behavior? and if it's not 0 it's default?